src/static/js/ace2_inner.js

```diff
@@ -2804,61 +2692,15 @@ function Ace2Inner() {
     // separated. If it returns null, it means that the list was not cut, try
     // from the current one.
     const line = caretLine();
-    if (line != -1 && renumberList(line + 1) === null) {
+    if (line !== -1 && renumberList(line + 1) == null) {
       renumberList(line);
     }
-  }
-
-  const REGEX_SPACE = /\s/;
+  };
 
   const isWordChar = (c) => padutils.wordCharRegex.test(c);
   editorInfo.ace_isWordChar = isWordChar;
 
-  function isSpaceChar(c) {
-    return !!REGEX_SPACE.exec(c);
-  }
-
-  function moveByWordInLine(lineText, initialIndex, forwardNotBack) {
-    let i = initialIndex;
-
-    function nextChar() {
-      if (forwardNotBack) return lineText.charAt(i);
-      else return lineText.charAt(i - 1);
-    }
-
-    function advance() {
-      if (forwardNotBack) i++;
-      else i--;
-    }
-
-    function isDone() {
-      if (forwardNotBack) return i >= lineText.length;
-      else return i <= 0;
-    }
-
-    // On Mac and Linux, move right moves to end of word and move left moves to start;
-    // on Windows, always move to start of word.
-    // On Windows, Firefox and IE disagree on whether to stop for punctuation (FF says no).
-    if (browser.msie && forwardNotBack) {
-      while ((!isDone()) && isWordChar(nextChar())) {
-        advance();
-      }
-      while ((!isDone()) && !isWordChar(nextChar())) {
-        advance();
-      }
-    } else {
```

Comment:
is this dead code? because this branch is executed when it's a non-ie browser

Reply:
probably a mistake!

Reply:
It's probably a mistake but afaik it was to support a function that was never included or contenteditable supported natively..

Reply:
I checked this and you are right. It was never used.